src/pages/OnboardingAccounting/BaseOnboardingAccounting.tsx

```diff
@@ -201,68 +194,11 @@ function BaseOnboardingAccounting({shouldUseNativeStyles}: BaseOnboardingAccount
 
-        if (!onboardingPurposeSelected || !onboardingCompanySize) {
-            return;
-        }
-
-        const shouldCreateWorkspace = !onboardingPolicyID && !paidGroupPolicy;
+        setOnboardingUserReportedIntegration(userReportedIntegration);
 
-        // We need `adminsChatReportID` for `completeOnboarding`, but at the same time, we don't want to call `createWorkspace` more than once.
-        // If we have already created a workspace, we want to reuse the `onboardingAdminsChatReportID` and `onboardingPolicyID`.
-        const {adminsChatReportID, policyID} = shouldCreateWorkspace
-            ? createWorkspace(undefined, true, '', generatePolicyID(), CONST.ONBOARDING_CHOICES.MANAGE_TEAM, '', undefined, false, onboardingCompanySize, userReportedIntegration)
-            : {adminsChatReportID: onboardingAdminsChatReportID, policyID: onboardingPolicyID};
-
-        if (shouldCreateWorkspace) {
-            setOnboardingAdminsChatReportID(adminsChatReportID);
-            setOnboardingPolicyID(policyID);
-        }
-
-        completeOnboarding({
-            engagementChoice: onboardingPurposeSelected,
-            onboardingMessage: onboardingMessages[onboardingPurposeSelected],
-            adminsChatReportID,
-            onboardingPolicyID: policyID,
-            companySize: onboardingCompanySize,
-            userReportedIntegration,
-        });
-
-        if (shouldOnboardingRedirectToOldDot(onboardingCompanySize, userReportedIntegration)) {
-            if (CONFIG.IS_HYBRID_APP) {
-                return;
-            }
-            openOldDotLink(CONST.OLDDOT_URLS.INBOX, true);
-        }
-
-        // Avoid creating new WS because onboardingPolicyID is cleared before unmounting
-        InteractionManager.runAfterInteractions(() => {
-            setOnboardingAdminsChatReportID();
-            setOnboardingPolicyID();
-        });
-
-        // We need to wait the policy is created before navigating out the onboarding flow
-        navigateAfterOnboardingWithMicrotaskQueue(
-            isSmallScreenWidth,
-            isBetaEnabled(CONST.BETAS.DEFAULT_ROOMS),
-            policyID,
-            adminsChatReportID,
-            // Onboarding tasks would show in Concierge instead of admins room for testing accounts, we should open where onboarding tasks are located
-            // See https://github.com/Expensify/App/issues/57167 for more details
-            (session?.email ?? '').includes('+'),
-        );
-    }, [
-        isBetaEnabled,
-        isSmallScreenWidth,
-        onboardingAdminsChatReportID,
-        onboardingCompanySize,
-        onboardingMessages,
-        onboardingPolicyID,
-        onboardingPurposeSelected,
-        paidGroupPolicy,
-        session?.email,
-        translate,
-        userReportedIntegration,
-    ]);
+        // Navigate to the next onboarding step with the selected integration
+        Navigation.navigate(ROUTES.ONBOARDING_INTERESTED_FEATURES.getRoute(route.params?.backTo));
```

Comment:
This caused a bug - https://github.com/Expensify/App/issues/67008#issuecomment-3110114182